jenkins-pipelines/manager/centos-manager-backup-200gb.jenkinsfile

```diff
@@ -5,9 +5,9 @@ def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
 
 managerPipeline(
     backend: 'aws',
-    region: '''["us-east-1", "us-west-2"]''',
+    region: 'us-west-2',
```

Comment:
it's on purpose you are putting each case on a different region ?

(if you can, you can even use `random`, I've fixed to be working a while back)

Reply:
I randomly chose one of the regions supported before in multiDC runs: "us-east-1" or "us-west-2"

Reply:
Just to explain, setting the region as random, can help a bit with the spot issues, since tests would spread to all of the supported regions, and won't all run on the exact same ones